modules/common/job/job.go

```diff
@@ -103,10 +127,20 @@ func (j *Job) DoJob(
 		j.changed = true
 	}
 
+	// NOTE(gibi): This should be in NewJob but then the defaulting would affect
```

Comment:
could reset the TTL for the object we pass to calc the hash and restore afterwards to not have an affect on the hash.

Reply:
right its complexity at a different place. I should have done that hash calc in NewJob right from the beginning. I can follow up doing that later with providing patches to the other operators.